apps/workflows/jobs/refill-daily.ts

```diff
@@ -13,12 +13,16 @@ client.defineJob({
   }),
 
   run: async (_payload, io, _ctx) => {
+    const date = new Date(Date.now());
+    const today = date.getDate();
     const db = connectDatabase();
     const tb = new Tinybird(env().TINYBIRD_TOKEN);
-    const t = new Date();
-    t.setUTCHours(t.getUTCHours() - 24);
+    const tDay = new Date();
```

Comment:
try to reuse an existing date reference you have
what if line 16 executes at midnight and line 21 executes slightly after?

Reply:
Think I solved it.